jax/third_party/numpy/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .linalg import *
+__all__ = ["cond", "tensorinv"]
```

Comment:
Is there a particular reason why we need this? `jax.third_party.numpy.cond` might be a little confusing. I would lean towards leaving this part out. I think `__init__.py` here could safely be empty.

Reply:
What was going on here was that I was importing everything into `jax/numpy/linalg.py` to make sure it's all in the right namespace. So I set that up such that the line in the JAX-native `linalg.py` so that things could just be imported with a wildcard and not necessarily drag all of the helper functions in, too. No worries though, this change has been made.